pages/04.HDM Doc 2.2/14.HDM reset/default.md

```diff
@@ -37,6 +37,7 @@ Pre-requisites
 
 
 1. HDM appliance must be powered on.
+2. In case of deployment modes other than Lite and Ultra Lite, for HDM reset to start, administrators will have to restart the praapa IOFilter service on all ESXi of the cluster on on-prem and then reboot the appliance manually.
```

Comment:
- Isn't this change required in macro as well, as with migration pages ?
- Also there is comment in the review google doc for this, so has that been verified ?

Reply:
The steps for HDM Reset are given in this page only.
And i have made some changes, please check now.

Reply:
Ok, fine.